app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors import NotVaccinatedError, OutdatedVaccineError

```

Comment:
You can import classes from the same module like this (check your `checklist` file in this task):
```
from app.errors import (
   NotVaccinatedError,
   OutdatedVaccineError,
   NotWearingMaskError
)
```

Reply:
ok